blogs/2025/05/27/ai-and-remote.md

```diff
@@ -0,0 +1,130 @@
+
+If you [publish](https://containers.dev/collections) any dev container configurations for others to use, we recommend exploring how you might add custom instructions to them! [Here](https://github.com/devcontainers/features/blob/main/src/python/devcontainer-feature.json#L80) is an example of using custom instructions in a Python dev container Feature:
+
+```json
+"github.copilot.chat.codeGeneration.instructions": [
+    {
+        "text": "This dev container includes `python3` and `pip3` pre-installed and available on the `PATH`, along with the Python language extensions for Python development."
+    }
+],
+```
+
+### Remote - SSH participant
+
+_Chat participants_ enhance your chat experience by providing domain-specific knowledge, such as how to interact with a database or a specific API. Type `@` in the chat input field to view and select from the list of available participants - they'll be available in Ask and Edit modes. There are several built-in chat participants like `@workspace`, `@vscode`, `@terminal`, and `@github`. Extensions can also contribute chat participants.
+
+The Remote - SSH extension includes a Copilot chat participant. Ask `@remote-ssh` for help configuring or troubleshooting elements of your remote environment:
+
+![Using Remote - SSH participant in VS Code chat panel](remote-ssh-intro.png)
+
+The participant is also knowledgeable about general SSH and remote development topics:
+
+![Asking Remote - SSH participant about remote auth](remote-ssh-auth.png)
+
+If a connection failure occurs, the **Diagnose with Copilot** feature provides a quick way to understand the problem:
+
+![Diagnose with Copilot option in notification](ssh-diagnose.png)
+
+Copilot will investigate the issue and provide actionable insights:
+
+<video src="analyzing.mp4" title="Copilot diagnosing connection issue" autoplay muted controls></video>
+
+### Manage tool approvals: Auto-approve agent mode tools
+
+As part of completing the tasks for a user prompt, [agent mode](/docs/copilot/chat/chat-agent-mode.md) can run tools and terminal commands. This is powerful but potentially comes with risks. Therefore, you need to approve the use of tools and terminal commands in agent mode.
+
+![Agent mode tool approval options dropdown](/release-notes/images/1_99/chat-tool-approval.png)
+
+In case you want to auto-approve _all_ tools, you can now use the experimental `chat.tools.autoApprove` setting. This will auto-approve all tools, and VS Code will not ask for confirmation when a language model wishes to run tools. This can help save time by giving more autonomy to agent mode, but bear in mind that with this setting enabled, you will not have the opportunity to cancel potentially destructive actions a model wants to take.
```

Comment:
Maybe bring the value prop of running in an isolated env forward? 

"To give you some level of protection against inadvertent terminal commands or tool calls, run it in a dev container, or remote machine to prevent your local dev machine from getting affected".

"In addition, this also enables you to take advantage of tool auto-approvals. ..."

Reply:
I like this idea! Grabbed your first suggestion and restructured a bit.